### ISSUE TITLE
Steam Workshop Publication: Technical Requirements

### ISSUE THREAD

**Issue by karlthepagan** [ScreepsOCS/screeps.behaviour-action-pattern#941]
How do we publish? What should the code look like? Where will it show up in the steam client?

**Comment by logxen** [comment]
It would end up here http://steamcommunity.com/workshop/browse?appid=464350

**Comment by Dasku** [comment]
Agreement is here: http://store.steampowered.com/subscriber_agreement/

Can't find where you actually submit. It looks like its an npm from the screenshots they provided in the news article here: http://steamcommunity.com/games/464350/announcements/detail/653414355541934468

**Comment by logxen** [comment]
yes you use npm. the process is documented here (scroll down): https://github.com/screeps/screeps

**Comment by karlthepagan** [comment]
The workshop item is a server-side mod. This issue now requires automation to complete.

After the NPM upload you submit the workshop mod via a submit form inside the steam server.

This also means owning a copy of the steam edition.
